static/app/views/dashboardsV2/create.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+
+import Feature from 'app/components/acl/feature';
+import Alert from 'app/components/alert';
+import {t} from 'app/locale';
+import {PageContent} from 'app/styles/organization';
+import {Organization} from 'app/types';
+import withOrganization from 'app/utils/withOrganization';
+
+import {EMPTY_DASHBOARD} from './data';
+import DashboardDetail from './detail';
+import {DashboardState} from './types';
+import {cloneDashboard} from './utils';
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  organization: Organization;
+  children: React.ReactNode;
+};
+
+function CreateDashboard(props: Props) {
+  function renderDisabled() {
```

Comment:
I noticed that you have two similar `renderDisabled`. Should we create a component for and reuse it?

Reply:
This is the only comment I didn't address- I don't know if it's worth it to have a bunch of nesting to abstract away of the duplication.